conf/gulp-tasks/templates.gulpfile.js

```diff
@@ -23,32 +23,38 @@ async function createTranslator (locale) {
  * Precompiles templates together, bundles them together,
  * cleans up any intermediary files, and kicks off the watch task.
  *
+ * @param {boolean} shouldWatch Whether or not to watch for changes. Defaults to true
  * @returns {Promise<Function>}
  */
-async function devTemplates () {
+async function devTemplates (shouldWatch = true) {
   const bundleTemplatesUMD =
     new BundleTemplatesTaskFactory(DEFAULT_LOCALE).create(TemplateType.UMD);
   const cleanFiles = createCleanFilesTask(DEFAULT_LOCALE);
   const translator = await createTranslator(DEFAULT_LOCALE);
   const precompileTemplates = createPrecompileTemplatesTask(DEFAULT_LOCALE, translator);
 
+  const nonWatchTasks = [precompileTemplates, bundleTemplatesUMD, cleanFiles];
```

Comment:
I updated `nonWatchTasks` to `templateTasks` and spread it in line 41 and simplified some of the promise logic. I don't think we need to split it into `devTemplates` and `basicTemplates` because this is similar to the pattern lower in `templates.gulpfile.js` where the locales build and search-bar-only build both use the same function, just with a differing boolean param passed in. but, if you feel strongly about it, I can change it